### ISSUE TITLE
Errors in tests when building jodconverter 4.2.2 with Java 9+

### ISSUE THREAD

**Issue by sysadm-prs-de** [sbraconnier/jodconverter#159]
I am rrying to build jodconverter 4.2.2 under MacOS 10.14.6 an get errors in the tests. Here is the log:

```
wwkempro:jodconverter 551$ gradle build -x test
> Task :jodconverter-local:integrationTest

org.jodconverter.filter.DefaultFilterChainITest > reset_WithEndsWithRefreshFilterOff_ShoudNotApplyRefreshFilter FAILED
    java.lang.NoSuchFieldException at DefaultFilterChainITest.java:156

org.jodconverter.filter.DefaultFilterChainITest > reset_WithEndsWithRefreshFilterOn_ShoudApplyRefreshFilter FAILED
    java.lang.NoSuchFieldException at DefaultFilterChainITest.java:156
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.assertj.core.util.introspection.FieldUtils (file:/Users/wolfgang/.gradle/caches/modules-2/files-2.1/org.assertj/assertj-core/3.11.1/fdac3217b804d6900fa3650f17b5cb48e620b140/assertj-core-3.11.1.jar) to field java.util.concurrent.atomic.AtomicInteger.value
WARNING: Please consider reporting this to the maintainers of org.assertj.core.util.introspection.FieldUtils
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

90 tests completed, 2 failed

> Task :jodconverter-local:integrationTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':jodconverter-local:integrationTest'.
> There were failing tests. See the report at: file:///Users/wolfgang/projects/setup/cms/src/xx/jodconverter/jodconverter-local/build/reports/integTest/index.html

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.5.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 3m 33s
54 actionable tasks: 1 executed, 53 up-to-date
```

Do I need to investigate or are these problems known and can be ignored? I have attached the protocol files for convenience. Any hints are very welcome. My goal is to replace jodconverter-2.2.2 in an old app that may benefit from an update.

[integTest.zip](https://github.com/sbraconnier/jodconverter/files/3461351/integTest.zip)

**Comment by axelrose** [comment]
same here.

I found that LibreOffice 6.2 is crashing while doing the tests

**Comment by sbraconnier** [comment]
Fixed by 6941199a9f0294a216aad099f3fdf904575d1d5d
